stdlib/Random/docs/src/index.md

```diff
@@ -198,7 +198,28 @@ Given a collection type `S`, it's currently assumed that if `rand(::S)` is defin
 
 #### Generating values for an `AbstractFloat` type
 
-`AbstractFloat` types are special-cased, because by default random values are not produced in the whole type domain, but rather in `[0,1)`. The following method should be implemented for `T <: AbstractFloat`: `Random.rand(::AbstractRNG, ::Random.SamplerTrivial{Random.CloseOpen01{T}})`
+`AbstractFloat` types are special-cased, because by default random values are not produced in the whole type domain, but rather in `[0,1)`. The following method should be implemented for `T <: AbstractFloat`: `Random.rand(::AbstractRNG, ::Random.SamplerTrivial{Random.CloseOpen01{T}})`.
+
+As an example:
+```jldoctest
+julia> struct Point{T} <: AbstractFloat
```

Comment:
That's somehow surprising to have `Point <: AbstratFloat`.

Reply:
I only used that so the example shows how to maybe generate a random floating-point number for a type with multiple feeds.

Reply:
You have a better suggestion for this?

Reply:
Check out the current example. I think it is okay.

Reply:
Just make it `MyFloat64` or something, this is IMO equally strange as the `Point` example.